lib/main.dart

```diff
@@ -15,51 +12,15 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: Constants.appName,
       theme: ThemeData(
         useMaterial3: true,
-        colorScheme: ColorScheme.fromSeed(seedColor: Colors.redAccent),
-      ),
-      home: const MyWidget(),
-    );
-  }
-}
-
-class MyWidget extends StatelessWidget {
-  const MyWidget({super.key});
-
-  @override
-  Widget build(BuildContext context) {
-    return Scaffold(
-      backgroundColor: Colors.grey,
-      body: SafeArea(
-        child: ListView(
-          children: const [
-            Center(
-              child: PosterImages(),
-            ),
-            Column(
-              children: [
-                MovieHeader(),
-                RatingStars(),
-                Padding(
-                  padding: EdgeInsets.all(Constants.descriptionPaddingSize),
-                  child: Text(
-                    Constants.overview,
-                    style: TextStyle(
-                      fontSize: Constants.descriptionTextSize,
-                      color: Colors.black,
-                    ),
-                  ),
-                ),
-                Center(
-                  child: LikeButton(),
-                )
-              ],
-            ),
-          ],
-        ),
+        colorScheme: ColorScheme.fromSeed(seedColor: Colors.blue),
       ),
+      initialRoute: MovieListScreen.routeName,
+      routes: {
+        MovieListScreen.routeName: (context) => const MovieListScreen(),
+        DetailScreen.routeName: (context) => const DetailScreen(),
```

Comment:
Add all the route in the same file like RoutesNames

Reply:
Done